src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -2924,27 +2924,138 @@ LogicalResult ONNXSplitV11Op::inferShapes(
 //===----------------------------------------------------------------------===//
 
 LogicalResult ONNXSplitToSequenceOp::verify() {
-  ONNXSplitToSequenceOpAdaptor operandAdaptor(*this);
-  Value input = operandAdaptor.input();
-  if (!hasShapeAndRank(input))
+  Value inputValue = input();
+  if (!hasShapeAndRank(inputValue))
     return success(); // Won't be able to do any checking at this stage.
 
-  auto inputType = input.getType().cast<ShapedType>();
-  int64_t inputRank = inputType.getShape().size();
-  int64_t axisIndex = axis();
+  ShapedType inputType = inputValue.getType().cast<ShapedType>();
+  auto inputShape = inputType.getShape();
```

Comment:
According to [LLVM coding standards](https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable), we can use `auto` with `cast<type>()` and should avoid `auto` in `auto inputShape = inputType.getShape();` I think `getShape` returns `ArrayRef<int64_t>`

Reply:
done

thanks for the explanation